pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -211,6 +211,9 @@ func generateVolumeMsg(prefixMsg, suffixMsg, volumeName, details string) (simple
 
 // VolumeToAttach represents a volume that should be attached to a node.
 type VolumeToAttach struct {
+	// AllowMultiAttach shows whether the volume is allowed to multi-attach
+	AllowMultiAttach bool
```

Comment:
Let's rename this. The name and description here are misleading. It doesn't control whether the volume allows or does not allow multiattach. Rather, when a volume is blocked due to multi-attach this field is used to prevent an event from being generated more than once. So how about calling it `MultiAttachErrorReported`?